src/components/CompareResults/ResultsTable.tsx

```diff
@@ -124,19 +124,28 @@ export default function ResultsTable() {
 
   return (
     <Box data-testid='results-table' role='table' sx={{ paddingBottom: 3 }}>
-      <ResultsControls
-        initialSearchTerm={initialSearchTerm}
-        frameworkId={frameworkIdVal}
-        resultsPromise={resultsPromise}
-        onSearchTermChange={onSearchTermChange}
-        onFrameworkChange={onFrameworkChange}
-      />
-      <TableHeader
-        cellsConfiguration={cellsConfiguration}
-        filters={tableFilters}
-        onToggleFilter={onToggleFilter}
-        onClearFilter={onClearFilter}
-      />
+      <Box
+        sx={{
+          position: 'sticky',
+          top: 0,
+          zIndex: 100,
+          bgcolor: 'background.default',
```

Comment:
this adjusts automatically with the dark mode thanks to the palette defined in https://github.com/mozilla/perfcompare/blob/0668d541c997bcedf1a5b43538141d786ac7e32d/src/theme/protocolTheme.ts#L11-L13 and https://github.com/mozilla/perfcompare/blob/0668d541c997bcedf1a5b43538141d786ac7e32d/src/theme/protocolTheme.ts#L29-L31